### PR TITLE
fix: remove ref for v4.loopback.io

### PR DESCRIPTION
**Pull request by dhmlau**
Remove any references for `v4.loopback.io`.  It includes:
- docs
- examples
- cli template

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: docs/site/deployment/Deploying_to_ibm_cloud_kubernetes.md [hunk 1]
```diff
@@ -9,8 +9,8 @@ permalink: /doc/en/lb4/deploying_to_ibm_cloud_kubernetes.html
 
 ## Overview
 
-This tutorial shows how to deploy a [LoopBack4](http://v4.loopback.io/)
-application onto [Kubernetes](http://kubernetes.io/) on the
+This tutorial shows how to deploy a [LoopBack4](http://loopback.io/) application
```
Comment (achrinza) [inline docs/site/deployment/Deploying_to_ibm_cloud_kubernetes.md]:
```suggestion
This tutorial shows how to deploy a [LoopBack 4](http://loopback.io/) application
```

File: examples/access-control-migration/public/index.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="shortcut icon" type="image/x-icon" href="//v4.loopback.io/favicon.ico">
+  <link rel="shortcut icon" type="image/x-icon" href="//loopback.io/favicon.ico">
```
Comment (achrinza) [inline examples/access-control-migration/public/index.html]:
```suggestion
  <link rel="shortcut icon" type="image/x-icon" href="https://loopback.io/favicon.ico">
```

File: examples/todo-list/public/index.html [hunk 3]
```diff
@@ -7,7 +7,7 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="shortcut icon" type="image/x-icon" href="//v4.loopback.io/favicon.ico">
+  <link rel="shortcut icon" type="image/x-icon" href="//loopback.io/favicon.ico">
```
Comment (achrinza) [inline examples/todo-list/public/index.html]:
```suggestion
  <link rel="shortcut icon" type="image/x-icon" href="https://loopback.io/favicon.ico">
```

File: examples/todo/public/index.html [hunk 4]
```diff
@@ -7,7 +7,7 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="shortcut icon" type="image/x-icon" href="//v4.loopback.io/favicon.ico">
+  <link rel="shortcut icon" type="image/x-icon" href="//loopback.io/favicon.ico">
```
Comment (achrinza) [inline examples/todo/public/index.html]:
```suggestion
  <link rel="shortcut icon" type="image/x-icon" href="https://loopback.io/favicon.ico">
```

File: packages/cli/generators/app/templates/public/index.html.ejs [hunk 5]
```diff
@@ -7,7 +7,7 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="shortcut icon" type="image/x-icon" href="//v4.loopback.io/favicon.ico">
+  <link rel="shortcut icon" type="image/x-icon" href="//loopback.io/favicon.ico">
```
Comment (achrinza) [inline packages/cli/generators/app/templates/public/index.html.ejs]:
```suggestion
  <link rel="shortcut icon" type="image/x-icon" href="https://loopback.io/favicon.ico">
```

File: docs/site/deployment/Deploying_to_ibm_cloud_kubernetes.md [hunk 6]
```diff
@@ -9,8 +9,8 @@ permalink: /doc/en/lb4/deploying_to_ibm_cloud_kubernetes.html
...
+onto [Kubernetes](http://kubernetes.io/) on the
```
Comment (achrinza) [inline docs/site/deployment/Deploying_to_ibm_cloud_kubernetes.md]:
```suggestion
onto [Kubernetes](https://kubernetes.io/) on the
```

File: examples/soap-calculator/public/index.html [hunk 7]
```diff
@@ -7,7 +7,7 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="shortcut icon" type="image/x-icon" href="//v4.loopback.io/favicon.ico">
+  <link rel="shortcut icon" type="image/x-icon" href="//loopback.io/favicon.ico">
```
Comment (achrinza) [inline examples/soap-calculator/public/index.html]:
```suggestion
  <link rel="shortcut icon" type="image/x-icon" href="https://loopback.io/favicon.ico">
```

File: docs/site/deployment/Deploying_to_ibm_cloud_kubernetes.md (hunk 1 above)
Comment (dhmlau) [reply docs/site/deployment/Deploying_to_ibm_cloud_kubernetes.md]: Thanks @achrinza. Perhaps we should change to use `https://loopback.io` as the url everywhere?

Comment (achrinza) [reply docs/site/deployment/Deploying_to_ibm_cloud_kubernetes.md]: Sounds like a good idea @dhmlau

**Comment by dhmlau** [comment]
@dougal83, when we're working on the rework of the web site for LoopBack 4, we created `v4.loopback.io` repo (and the url) as an interim solution for keeping both LB3 and LB4 info. With LB4 being the current release, that's why we're moving the content back to loopback.io.  Supposedly v4.loopback.io shouldn't be reference, IMHO.

**Comment by dhmlau** [comment]
@dougal83, are you good if I land this PR? Thanks.

**Pull request merged**